gpu_ocean/SWESimulators/BaseOceanStateEnsemble.py

```diff
@@ -85,10 +114,18 @@ def cleanUp(self):
             self.observation_buffer.release()
         self.gpu_ctx = None
         
-    # IMPROVED
     def setGridInfo(self, nx, ny, dx, dy, dt, 
                     boundaryConditions=Common.BoundaryConditions(), 
                     eta=None, hu=None, hv=None, H=None):
+        warnings.warn('The function setGridInfo will be deprecated. Please use the improved constructor instead',
```

Comment:
This should probably be "is deprecated" or "will be removed". Marking functions for later removal _is_ deprecation. Goes for all instances of DeprecationWarning in the PR.